flask-vue/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:latest
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+RUN apt-get update && \
+    apt-get install -y nodejs npm &&\
+    rm -rf /var/lib/apt/lists/*
```

Comment:
The propose is reduce the size of a docker image.
That command deletes the lists of packages downloaded by apt-get update